### PR TITLE
fix(search): added aria-label

### LINKED ISSUE #1739: Accessibility Issue: <ibm-search>
## Detailed description

> Describe in detail the issue you're having. 
> 
If you use two ibm search components on the same page, you get an accessibility violation from IBM's accessibility plugin about "Multiple elements with "search" role do not have unique labels" and the only fixes are to add the aria-label or aria-labeledby attributes to the elements with the "search" role.
Edit: issue also happens if you have one `<ibm-search>` and one `<ibm-table-toolbar-search>`
> Is this a feature request (new component, new icon), a bug, or a general issue?
>
General Issue
> Is this issue related to a specific component?
>
Search component, `<ibm-search>`
> What did you expect to happen? What happened instead? What would you like to see changed?
>
Arialabel input needs to be added and binded to the [div with the 'search' role](https://github.com/IBM/carbon-components-angular/blob/a4953f2de3b2b3b37541978ea65cb589313005cd/src/search/search.component.html#L1-L14)
> What browser are you working in?
>
Chrome
> What version of the Carbon Design System are you using?
>
v4
>  What offering/product do you work on? Any pressing ship or release dates we should be aware of?
>
MCMP
## Steps to reproduce the issue
1. Install the [IBM accessibility checker](https://www.ibm.com/able/toolkit/tools/)
2. Run it on a webpage with two <ibm-search> components or one `<ibm-search>` and one `<ibm-table-toolbar-search>
`

### PR DESCRIPTION
**Pull request by amchris98**
Fixes #1739

### CODE REVIEW

File: src/search/search.component.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 		'bx--toolbar-search--active': toolbar && active
 	}"
 	role="search"
+	aria-label="ariaLabel"
```
Comment (zvonimirfras) [inline src/search/search.component.html:13]: This puts literal string `"ariaLabel"` for aria-label. It should probably be something like 
```suggestion
	[attr.aria-label]="ariaLabel"
```
to make it take the input value.

**Review by zvonimirfras — CHANGES_REQUESTED** [review]
Thanks for the quick turnaround. Some changes requested :)

File: src/search/search.component.html (hunk 1 above)
Comment (amchris98) [reply src/search/search.component.html]: oops my bad, muscle memory has me used to feeding an aria-label as an input to other components.

**Review by cal-smith — APPROVED** [review]
LGTM! Thanks for this! 🚀

**Review by zvonimirfras — APPROVED** [review]

**Pull request merged**